cve_bin_tool/vex_manager/generate.py

```diff
@@ -134,7 +134,9 @@ def get_vulnerabilities(self) -> List[Vulnerability]:
                 vulnerability.set_status(self.analysis_state[self.vextype][cve.remarks])
                 if cve.justification:
                     vulnerability.set_justification(cve.justification)
-                # vulnerability.set_remediation(cve.response)
+                # set_remediation is not available for lib4sbom pypi version for now
+                # if cve.response:
+                # vulnerability.set_remediation(cve.response[0])
```

Comment:
The vulnerability class has a set_value function which allows the setting of any attribute. 

e.g. `vulnerability.set_value("remediation","Some interesting remediation description")`

Reply:
@anthonyharrison i thought the same of using set_value as the Vulnerability class do not include set_remediation in pypi version but even if set it the cyclonedx generator in lib4sbom latest pypi version do not take remediation into account for now so i thought to use it in comment, although i am aware that csaf takes it into account,  I'll shift it to set_value()